src/app/file-manager/file-explorer/file-explorer.component.ts

```diff
@@ -24,13 +24,16 @@ export class FileExplorerComponent implements OnInit {
   msgs: Message[];
 
   @Output('done') done: EventEmitter<any> = new EventEmitter();
```

Comment:
Is `'done'` connected to any `@Input`? I don't see a reference to it from another component.

Reply:
Yes, it is used in home.component.html component to change the tab.